zio-http/jvm/src/test/scala/zio/http/endpoint/openapi/OpenAPIGenSpec.scala

```diff
@@ -2812,6 +2812,13 @@ object OpenAPIGenSpec extends ZIOSpecDefault {
             |          "nestedOption" : {
             |            "$ref" : "#/components/schemas/Recursive"
             |          },
+            |          "nestedMap" : {
+            |            "type" : "object",
+            |            "properties" : {},
+            |            "additionalProperties" : {
+            |              "$ref" : "#/components/schemas/Recursive"
+            |            }
+            |          },
```

Comment:
@hochgi @987Nabil I checked with the `openapi-generator-cli validate` cli and this definition is valid 🙂

Reply:
There's no need for `"properties" : {}`, its redundant. But no harm either I guess :)

Reply:
Yes, it's not but I don't know how to remove it from the generated spec and the spec is valid with it, so yeah, no harm 🙂